dev-docs/RFCs/v7.1/effects-module-rfc.md

```diff
@@ -0,0 +1,69 @@
+# RFC: @luma.gl/effects Module
+
+* **Authors**: Jian Huang
+
+* **Date**: May. 2019
+
+* **Status**: For Review
+
+## Abstract
+A lot of shader modules focusing on effect rendering are created, better to have a sub-module for them instead of scattered all over the place.
+
+## Status Quo
+* Source code structure for effects in `@luma.gl/glfx`
+	<pre>
+	glfx/src
+		|
+		adjust-filters
+		|
+		blur-filters
+		|
+		curve-filters
+		|
+		fun-filters
+		|
+		warp-filters
+	</pre>
+* Source code structure for effects in current `@luma.gl/effects`
+    <pre>
+	 effects/src
+		    |
+		    modules
+		        |
+		        convolution.js
+		        |
+		        depth.js
+		        |
+		        pack.js
+		        |
+		        ssao.js
+		    |
+		    passes
+		        |
+		        convolution-pass.js
+		        |
+		        outline-pass.js
+		        |
+		        ssao-pass.js
+    </pre>
+			
+## Proposal
+New source code structure in `@luma.gl/effects`
+<pre>
+effects/src
+	|
+	modules
+	    |
+	    filter folders from `@luma.gl/gflx`
+	|
+	experimental
+	    |
+	    folders from old `@luma.gl/effects`
+</pre>
+
+* Objects under experimental folder will be exported as `_ObjectName`
+* `@luma.gl/glfx` will re-export necessary objects from @luma.gl/effects for backward compatibility
```

Comment:
I propose we drop the `@luma.gl/glfx` module entirely in 7.1. If no new release is published for this submodule, no backward compatibility is needed.